service/controller/app/v1/resource/chart/resource.go

```diff
@@ -32,19 +39,32 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 	return nil, nil
 }
 
-func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interface{}, error) {
-	return nil, nil
-}
-
 func (r *Resource) NewUpdatePatch(ctx context.Context, obj, currentState, desiredState interface{}) (*controller.Patch, error) {
-	return nil, nil
+	patch := controller.NewPatch()
+	patch.SetCreateChange(desiredState)
```

Comment:
Why is this set?

In a later PR `SetCreateChange` can be implemented but this is incorrect as it doesn't consider the current state.

Reply:
I set this method just for checking the creation of `chart` CRD, I can comment it out now rather than delete it, What do you think?

Reply:
I'd prefer it was removed but in this case I guess its OK.

I'm not a fan of lots of commented out code. It's untidy and its easy to forget to update it later.

Reply:
In this case I need to fill out these functions eventually :)

Reply:
>  but in this case I guess its OK

Yep :) but still not a fan of this approach.